app/filtration/filtrationPage.tsx

```diff
@@ -71,14 +119,51 @@ export default function FiltrationPage(): React.JSX.Element {
       <ScrollView
         style={styles.scrollView}
         contentContainerStyle={styles.contentContainer}
-        showsVerticalScrollIndicator={true}>
+        showsVerticalScrollIndicator={true}
+        scrollEnabled={scrollEnabled}>
         <View style={styles.topSpacer} />
 
         <View style={styles.filtrationsContainer}>
-          <View style={styles.filtrationContainer}>
-            <Text style={styles.filtrationTitle}>ToDo</Text>
-            <Text style={styles.filtrationSubtitle}>ToDo</Text>
-          </View>
+          <SexSelectorOtter
+            title="Płeć"
+            subtitle="Jakiej płci szukasz?"
+            value={selectedSex}
+            onChange={(newSex) => setSelectedSex(newSex)}
+            multiSelect={true}
+          />
+          <SliderOtter
+            title="Maksymalny Dystans"
+            subtitle="Do jakiej odległości wydra może odpłynąć?"
+            value={distanceRange}
+            onChange={(newDistance) => setDistanceRange(newDistance as number)}
+            minValue={1}
```

Comment:
```suggestion
            minValue={5}
```
biorąc pod uwage mechanizm zaciemniania lokalizacji imo nie ma sensu ustawiać tutaj dystansu mniejszego niż 5km. Ew. myślałem nawet o 10km jako wartość minimalna

Reply:
Aktualny wzór na maksymalny dystans dummyLocation można obliczyć tylko na podstawie initial_side_km. Więc możemy ustalić minimalną wartość względem initial_side_km i zaokrąglić do góry

initial_side_km = 2 (aktualnie tak mamy)
pierwszy_skok = initial_side_km * (initial_side_km - 1)
drugi_skok = initial_side_km / 2
maxDystansWLiniProstej = pierwszy_skok + drugi_skok 
maxDystans = sqrt((maxDystansWLiniProstej^2 + maxDystansWLiniProstej^2))

Czyli initial_side_km = 2
pierwszy_skok = 2 * (2 - 1) = 2
drugi_skok = 2 / 2 = 1
maxDystansWLiniProstej = 2 + 1 = 3
maxDystans = sqrt((3^2 + 3^2)) = sqrt(18) = 4.24km

Dla na przykład initial_side_km = 4
pierwszy_skok = 4 * (4 - 1) = 12
drugi_skok = 4 / 2 = 2
maxDystansWLiniProstej = 12 + 2 = 14
maxDystans = sqrt((14^2 + 14^2)) = sqrt(392) = 19.8km